### ISSUE TITLE
Serious issue on Anki 2.1.24beta4

### ISSUE THREAD

**Issue by AnKingMed** [cjdduarte/Free_Weekend#4]
This causes an issue on 2.1.24beta4 that breaks the profile and you have to restore from backup when you try to access the options for a deck

```
Debug info:
Anki 2.1.24 (16ab351b) Python 3.8.0 Qt 5.13.1 PyQt 5.14.1
Platform: Mac 10.15.4
Flags: frz=True ao=True sv=2
Add-ons, last update check: 2020-04-07 15:05:41

Caught exception:
Traceback (most recent call last):
  File "aqt/deckconf.py", line 323, in accept
  File "</Applications/Anki.app/Contents/MacOS/decorator.pyc:decorator-gen-154>", line 2, in saveConf
  File "anki/hooks.py", line 638, in decorator_wrapper
  File "anki/hooks.py", line 633, in repl
  File "</Applications/Anki.app/Contents/MacOS/decorator.pyc:decorator-gen-114>", line 2, in saveConf
  File "anki/hooks.py", line 638, in decorator_wrapper
  File "anki/hooks.py", line 633, in repl
  File "</Applications/Anki.app/Contents/MacOS/decorator.pyc:decorator-gen-46>", line 2, in saveConf
  File "anki/hooks.py", line 638, in decorator_wrapper
  File "anki/hooks.py", line 633, in repl
  File "aqt/deckconf.py", line 317, in saveConf
  File "anki/decks.py", line 77, in save
  File "anki/decks.py", line 342, in update_config
  File "anki/rsbackend.py", line 516, in add_or_update_deck_config
orjson.JSONEncodeError: Type is not JSON serializable: CheckState
```

**Comment by cjdduarte** [comment]
Thanks for the report. I will evaluate.

**Comment by AnKingMed** [comment]
Definitely possible things could change with future betas, but I wanted to give you a heads us especially since it causes such a serious issue

**Comment by cjdduarte** [comment]
As a precaution, I limited the installation of the addon to version 2.1.23.
But I will evaluate.

Tanks

**Comment by cjdduarte** [comment]
Which line of code did the error occur on?
I ask, because I use windows and installed version 24b4 in my development base and the addon is working.

**Comment by cjdduarte** [comment]
i found the error.

**Comment by AnKingMed** [comment]
Awesome! Happy to test in any way if you need.  Love the addon :)

**Comment by cjdduarte** [comment]
I temporarily disabled the functionality that caused the error. I will wait to see if it is not an error in this new beta or if it really is a definite structural change.

But Ad-don is functioning normally.
Thank you again.

**Comment by AnKingMed** [comment]
Awesome. Does that change the functionality of the add-on at all?  I try to test the betas and alert add-on authors so if I find any further issues I'll let you know

**Comment by cjdduarte** [comment]
No. This functionality in the deck settings was causing the error.
Few people used it. I temporarily disabled it.

![image](https://user-images.githubusercontent.com/42243532/78817956-4e1ac580-79aa-11ea-98bf-13dd3c136726.png)

**Comment by AnKingMed** [comment]
Ah smart. That makes sense. Thanks for the quick fix!

**Comment by cjdduarte** [comment]
v1.12 - 2020-04-10 - Bug fix: Correct incompatibility with version 2.1.24. Functionality enabled again
